backends/blocked/ceed-blocked.h

```diff
@@ -22,6 +22,7 @@ typedef struct {
 } CeedBasis_Blocked;
 
 typedef struct {
+  bool add;
```

Comment:
I added this same argument to the top of the Ref and Blocked struct to avoid duplicating the composite operator apply, but this may be fragile.

Reply:
@jedbrown this is the one thing I'd most want your opinion on before merging.